ecs-update/ecs-update.go

```diff
@@ -864,6 +864,9 @@ func Edit(pkg *fleetpkg.Integration, c EditConfig) (*EditResult, error) {
 		result: &EditResult{},
 	}
 
+	if pkg.Input != nil {
+		pkg.DataStreams["_input_root"] = pkg.Input
```

Comment:
> I don't like that this works, but it does.

This was also what I originally had in mind to make input type packages appear to be just like integration type packages.